test/services/models-manager.test.js

```diff
@@ -0,0 +1,69 @@
+const ModelsManager = require('../../src/services/models-manager');
+
+describe('services > models-manager', () => {
+  describe('getModels', () => {
+    const configStore = {
+      Implementation: {
+        getModels: jest.fn(() => ({
+          model1: {
+            name: 'model1',
+          },
+          model2: {
+            name: 'model2',
+          },
+        })),
+        getModelName: jest.fn((model) => model.name),
+      },
+    };
+
+    const modelsManager = new ModelsManager({ configStore });
+
+    it('should return model lists', () => {
```

Comment:
```suggestion
    it('should return the model lists as an object using model name as a key', () => {
```

(Had hard time figuring this out, maybe the label of the test was ok, but a small comment on the difference between `getModels` and `getModelArray`would be nice)